i18n/de.yaml

```diff
@@ -3,7 +3,7 @@
   translation: "Über Uns"
 
 - id: title_apps
-  translation: "Download" # TODO: translate
+  translation: "Herunterladen"
```

Comment:
Hm I'm not sure but we can probably use "Download" here. I feel like "Download" is already pretty commonly used in the German language like in "Hier geht's zum Download".

Reply:
Yes it is, and most people should indeed be able to understand it :+1: